.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,27 +1,30 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
-labels: bug
+title: "[BUG]"
+labels: 'Type: Bug'
```

Comment:
This is something I would like opinion on.

I like "[TITLE]" because it shows clearly that the user is using our template, and it's clear what the issue is about.
I like "Type: Issue" as a label, because sometimes issues have many tags, and this way it's easier to spot the Type and Priority, and other more important labels.

Reply:
Same for `Feature` then?

Reply:
NVM, you did 👌